.github/check_agda.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+export Agda_datadir=$PWD/data
+export PATH=$PWD/bin:$PATH
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  # See https://gitlab.haskell.org/ghc/ghc/-/issues/17418
+  xattr -rc $PWD/bin
+fi
+
+echo "The nightly version of Agda is"
+agda --version
+
+echo "Checking if Agda nightly is runnable on your system..."
+for prim in ./data/lib/prim/Agda/**/*.agda
+do
+    echo "Checking "$prim
+    agda -v 2 $prim
```

Comment:
Quoting
```diff
-echo "Checking "$prim
+echo "Checking ${pim}"
-agda -v 2 $prim
+agda -v2 "${prim}"
```

(The braces I added aren't necessary, just a habit for consistency).

Reply:
Done.